com.github.AberothCommunity.Aberoth.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: com.github.AberothCommunity.Aberoth
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk8
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --persist=Downloads
```

Comment:
This seems wrong, do you mean `--filesystem=xdg-download`?

Reply:
The game saves it's config and cache files in "~/Downloads/aberoth". This is my first flatpak but it seems to work correctly.

Reply:
https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#filesystem-permissions

Reply:
Ah ok, then it is correct. Downloads is the wrong location to save application data